scripts/build.py

```diff
@@ -1334,6 +1343,9 @@ def main(args=None):
         raise Exception(
             'maintenance_mode should only be enabled in prod build.')
 
+    # Clean up the existing generated folders.
+    clean()
```

Comment:
Should this rather be called always when we finish the setup. E.g. also in cases when the build script is cancelled by Ctrl+C or similar?

Reply:
What do you mean by finish the setup? I am not sure what's the setup stage for build.

Reply:
Ah sorry, I mean when we either finish the build or cancel the build script.

Reply:
Hmm, so we should not delete folders after build is finished, right?

Re, the cancel thing, clean up will be done automatically whenever we run it again and build has to be run as part of deployment. So, keeping it at the start handles that issue as well.

Reply:
Right, sorry, I got confused.